frontend/util.lua

```diff
@@ -150,7 +150,7 @@ end
 ---- @treturn string clock string in the form of 1h30'10'' or 1h30m10s
 function util.secondsToHClock(seconds, withoutSeconds, hmsFormat)
     -- @translators \" stands for the seconds symbol
-    SECONDS_SYMBOL = _("\"")
+    local SECONDS_SYMBOL = _("\"")
```

Comment:
Why make this one translatable but not the other (`'`)?

(Sidebar: making any of 'em translatable at all feels overkill to me).